docs/rst/formalia/titlepage.rst

```diff
@@ -0,0 +1,21 @@
+.. raw:: html
+
+  <h1>
+    Vulcanexus Documentation
+  </h1>
+
+.. image:: /rst/figures/logo.png
+  :height: 100px
+  :width: 100px
+  :align: left
+  :alt: Vulcanexus
+  :target: http://www.vulcanexus.com/
+
+.. Vulcanexus introduction
+
+.. main_feature:
```

Comment:
```suggestion
.. _main_feature:
```

Reply:
I would remove this comment as it is not possible to convert into a label and can lead to misunderstandings.